src/main/java/HW14/Fruits/Box.java

```diff
@@ -0,0 +1,51 @@
+package HW14.Fruits;
+
+import java.util.Collection;
+import java.util.LinkedList;
+
+public class Box<T extends Fruit>{
+    LinkedList<Fruit> box = new LinkedList<>();
+
+    public void addToBoxOne(Fruit fruit) {
```

Comment:
```suggestion
    public void add(T fruit) {
```

і тоді тіло методу буде: 
```
box.add(fruit);
```